### ISSUE TITLE
Feature request: log-log scale for tensorboard

### ISSUE THREAD

**Issue by nikitakit** [tensorflow/tensorflow#6532]
Would there be interest in adding a log-log display mode to tensorboard? That is, to enable plotting the x-axis in log scale for step count (and maybe relative time as well).

I find log-log to be very useful for inspecting learning curves (loss vs. iteration). Given a constant learning rate, standard optimization algorithms usually give a straight line in log-log scale. It's really easy to visually spot when the straight line behavior ends, which is generally a sign that the model has almost converged (for its current learning rate). On the other hand, in lin-lin you can't distinguish the two conditions because everything looks vaguely like exponential decay, and log-lin isn't ideal either.

**Comment by aselle** [comment]
@danmane, could you consider this feature?

**Comment by danmane** [comment]
Hi @nikitakit, thanks for the suggestion!

@bdilday sent a pull request a few months ago (#3876) which added y-axis log scales on a per chart basis. I think it shouldn't be too hard to add x-axis log scales either. The main question in my mind is how to incorporate it into the UI: should it be a per-chart switch (like the y-axis log scales) or a global switch? Right now we coordinate all the x-axes through the global switch on the left, so it might be best to add it through another global setting. (Although I'm reluctant to keep adding checkboxes as a UI paradigm, that might be the right way to accommodate this request.)

The core TensorBoard team has other priorities right now, so we won't likely add this ourselves, but given bdilday's PR as a template, it should be feasible for someone in the community to put this in. I'm marking the issue contributions welcome and un-assigning myself.

**Comment by finbarrtimbers** [comment]
I can take this on if no one else is working on it.

**Comment by danmane** [comment]
Go for it :)

**Comment by wchargin** [comment]
I'm closing this due to inactivity, but if you have a PR, please feel free to submit it to our new repository at https://github.com/tensorflow/tensorboard/pulls! Thanks :-)

**Comment by Remi-Coulom** [comment]
I would love an horizontal log scale, too.

**Comment by GeoffNN** [comment]
This would be awesome to check convergence rates.
